src/PHPStanAnalyser.php

```diff
@@ -4,168 +4,35 @@
 
 namespace Pest\TypeCoverage;
 
-use Composer\InstalledVersions;
 use PhpParser\Node;
 use PHPStan\Analyser\Analyser;
 use PHPStan\Analyser\FileAnalyser;
-use PHPStan\Analyser\IgnoreErrorExtensionProvider;
-use PHPStan\Analyser\LocalIgnoresProcessor;
 use PHPStan\Analyser\NodeScopeResolver;
-use PHPStan\Analyser\RuleErrorTransformer;
 use PHPStan\Collectors\Collector;
 use PHPStan\Collectors\Registry;
-use PHPStan\Dependency\DependencyResolver;
 use PHPStan\DependencyInjection\Container;
-use PHPStan\DependencyInjection\Reflection\ClassReflectionExtensionRegistryProvider;
-use PHPStan\DependencyInjection\Type\DynamicThrowTypeExtensionProvider;
-use PHPStan\DependencyInjection\Type\ParameterClosureTypeExtensionProvider;
-use PHPStan\DependencyInjection\Type\ParameterOutTypeExtensionProvider;
-use PHPStan\File\FileHelper;
-use PHPStan\Php\PhpVersion;
-use PHPStan\PhpDoc\PhpDocInheritanceResolver;
-use PHPStan\PhpDoc\StubPhpDocProvider;
-use PHPStan\Reflection\AttributeReflectionFactory;
-use PHPStan\Reflection\InitializerExprTypeResolver;
-use PHPStan\Reflection\ReflectionProvider;
-use PHPStan\Reflection\SignatureMap\SignatureMapProvider;
 use PHPStan\Rules\DirectRegistry;
-use PHPStan\Rules\Properties\ReadWritePropertiesExtensionProvider;
 use PHPStan\Rules\Rule;
-use PHPStan\Type\FileTypeMapper;
 
 /**
  * @internal
  */
 final class PHPStanAnalyser
 {
     /**
-     * Creates an analyser with the rules and collectors needed for type coverage.
+     * Creates an analyzer with the rules and collectors needed for type coverage.
```

Comment:
Why? the class name is `PHPStanAnalyser` (it uses `s` in `analyser`)

Reply:
Yeah, this change can be reverted as it's unnecessary. The other bits look good to me. 👍🏻